bee-node/README.md

```diff
@@ -9,22 +9,22 @@
 ```sh
 apt-get update
 apt-get upgrade
-apt-get install git npm build-essential cmake pkg-config librocksdb-dev llvm clang libclang-dev libssl-dev
```

Comment:
😍

Reply:
I don't know why I deleted this! Will add back all apart from `npm`.

Reply:
Oh wait, I didn't delete it (it's on the next line of the diff without `npm`, but this comment doesn't show it :trollface:)